chrome/content/zotero/integration/quickFormat.js

```diff
@@ -1105,6 +1241,17 @@ var Zotero_QuickFormat = new function () {
 			_refocusQfe();
 		}
 	}
+
+	function editorHeightLock(isLocked) {
```

Comment:
Good catch - I added some comments. 

This addresses that issue I brought up earlier where editor and window height can find themselves out of sync for a moment during drag-drop, and it was really bothering me. This forces the height of the editor to remain unchanged as bubbles are reshuffled to avoid it.